test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -180,3 +180,13 @@ Describe "Validate that Get-Help returns provider-specific help" -Tags @('CI', '
         }
     }
 }
+
+Describe "Validate about_help.txt under culture specific folder works" -Tags @('CI') {
+
+    It "Get-Help about_should should return help text and not multiple HelpInfo objects" {
+
+        $help = Get-Help about_should
```

Comment:
My thought was that if someone is going to have to improve Help System and to rename this file, the test will fail while has no relationship to the improvements.